projects/go-lib/src/lib/components/go-select/go-select.component.spec.ts

```diff
@@ -102,4 +102,19 @@ describe('GoSelectComponent', () => {
       expect(component.control.value).toEqual([1, 2, 3]);
     });
   });
+
+  describe('onRemoveAll()', () => {
```

Comment:
This should just be `onRemoveAll`, the parenthesis are not needed.

Reply:
Hi @grahamhency 
Removed parenthesis from onRemoveAll. Please check